content/blog/sprint-review-2010.md

```diff
@@ -0,0 +1,41 @@
+Title: Wazo Platform 20.10 Released
+Date: 2020-07-20
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2010
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 20.10 release.
+
+## Technical features
+
+* **Asterisk**: Asterisk was upgraded to 17.5.1. See the [Asterisk release announcement](https://www.asterisk.org/downloads/asterisk-news/asterisk-1751-now-available)
+* **Performance**: We have finished removing the use of encryption for all internal HTTP communication between Wazo components. Encryption now only happens when Wazo is sending/receiving data from the outside world.
+* **Multi-tenancy**: We have improved the consistency of the database for multiple tenants. When a tenant is removed, most of the data related to this tenant are now removed.
```

Comment:
Oh it's really not true. maybe less than 10% of data related to this tenant is removed. Maybe something like this:
```suggestion
* **Multi-tenancy**: We have fixed some consistency of the database for multiple tenants.
```

But IMO we should remove this one too

Reply:
Or be more specific.
`Users will now get deleted from the database after the tenant gets deleted.`

Reply:
I added Pascal's suggestion. Thanks!